biorbd_optim/objective_functions.py

```diff
@@ -103,6 +103,7 @@ class Lagrange(Enum):
         ALIGN_SEGMENT_WITH_CUSTOM_RT = (PenaltyType.ALIGN_SEGMENT_WITH_CUSTOM_RT,)
         ALIGN_MARKER_WITH_SEGMENT_AXIS = (PenaltyType.ALIGN_MARKER_WITH_SEGMENT_AXIS,)
         CUSTOM = (PenaltyType.CUSTOM,)
+        TRACK_CONTACT_FORCES = (PenaltyType.TRACK_CONTACT_FORCES,)
```

Comment:
Please move this right after TRACK_ALL_CONTROLS

Reply:
done

Reply:
done